tedana/selection/tedica.py

```diff
@@ -386,10 +386,10 @@ def kundu_selection_v2(comptable, n_echos, n_vols):
         unclf = np.setdiff1d(unclf, midk)
 
         # Find components to ignore
-        # Ignore high variance explained, poor decision tree scored components
-        new_varex_lower = stats.scoreatpercentile(
-            comptable.loc[unclf[:num_acc_guess], "variance explained"], LOW_PERC
-        )
+        # Of the remaining components, ignore ones with higher variance even if their
+        # decision tree scores are poor
+        sorted_varex = np.flip(np.sort((comptable.loc[unclf, "variance explained"]).to_numpy()))
```

Comment:
```suggestion
        sorted_varex = np.flip(np.sort(comptable.loc[unclf, "variance explained"].to_numpy()))
```

This shouldn't impact behavior, but I think it's more readable to get the numpy array, sort it, then flip it, instead of sorting the pandas Series, getting the numpy array from that, sorting it, and then flipping it.